lbz/authz/authorizer.py

```diff
@@ -132,8 +132,7 @@ def _check_allow_and_set_resources(self) -> None:
         ):
             return
         if d_domain := self.allow.get(self.resource):
-            if self._allow_if_allow_all(d_domain):
-                return
+            self._allow_if_allow_all(d_domain)
```

Comment:
I didn't check tests yet but it doesn't look like a good change. Consider the case hen `d_domain` is equal to `ALL ("*")`. In the line below, you will get `AttributeError` then 🤔

Reply:
Oh, the `self._allow_if_allow_all` is also used above 🤔  So, correct me if I am wrong but in my opinion, here, it shouldn't be triggered at all then.